pkg/operator/csi/csidrivernodeservicecontroller/csi_driver_node_service_controller_test.go

```diff
@@ -160,16 +160,6 @@ func withLogLevel(logLevel opv1.LogLevel) driverModifier {
 	}
 }
 
-func withGeneration(generations ...int64) driverModifier {
-	return func(i *fakeDriverInstance) *fakeDriverInstance {
-		i.Generation = generations[0]
-		if len(generations) > 1 {
-			i.Status.ObservedGeneration = generations[1]
```

Comment:
Yup, that's weird.

Reply:
It turns out nothing was really setting this field, we were just adding and confirming its presence in tests. :disappointed_relieved: